Python/ast_unparse.c

```diff
@@ -58,6 +58,12 @@ append_charp(_PyUnicodeWriter *writer, const char *charp)
         } \
     } while (0)
 
+#define INF_STR PyUnicode_FromString("inf")
+
+#define INF_VALUE PyUnicode_Concat(PyUnicode_FromString("1e"), \
```

Comment:
Every of these calls can raise an exception and return NULL. Also there are reference leaks here.

Reply:
:+1: